composer.json

```diff
@@ -19,7 +19,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": ">=5.3",
+        "php": "^5.5 || ^7.0",
```

Comment:
Using `||` here to specify alternate version requirements - we're not really sure yet whether this will support PHP 9000, for example, hehe!